src/BenchmarksDriver/Program.cs

```diff
@@ -444,6 +454,7 @@ public static int Main(string[] args)
                         [ConnectionFilter] [nvarchar](max) NULL,
                         [WebHost] [nvarchar](max) NOT NULL,
                         [KestrelThreadCount] [int] NULL,
+                        [KestrelThreadPoolDispatching] [bit] NULL,
```

Comment:
I understand your point, and these could have been designed either way.  But I think it's more flexible for the benchmarks automation to allow the caller to specify `kestrel default` (`null`) in addition to `true` and `false`.  It matters less for a boolean which must default to either `true` or `false`.  It matters more for a setting like `KestrelThreadCount` where the default is something computed dynamically (`NumCores / 2`).